libgoal/libgoal.go

```diff
@@ -468,6 +487,20 @@ func (c *Client) SendPaymentFromWallet(walletHandle, pw []byte, from, to string,
 // ConstructPayment builds a payment transaction to be signed
 // If the fee is 0, the function will use the suggested one form the network
 func (c *Client) ConstructPayment(from, to string, fee, amount uint64, note []byte, closeTo string) (transactions.Transaction, error) {
+	// Get current round, protocol, genesis ID
+	params, err := c.SuggestedParams()
+	if err != nil {
+		return transactions.Transaction{}, err
+	}
+	firstRound := basics.Round(params.LastRound)
```

Comment:
We might as well use `LastRound+1`, because `LastRound` is already committed, and there's no chance that this new transaction will end up in it.